jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/LambdaSamRunConfigurationProducer.kt

```diff
@@ -84,5 +94,21 @@ class LambdaSamRunConfigurationProducer : RunConfigurationProducer<SamRunConfigu
 
     companion object {
         private fun getFactory() = LambdaRunConfiguration.getInstance().configurationFactories.first { it is SamRunConfigurationFactory }
+        private fun settings(project: Project): Pair<String?, AwsRegion?> {
```

Comment:
Can we rename this to something like `profileSettings` as it only returns the current account and region?
`settings` seems too generic to me .

Reply:
done!